Solutions/CTM360/Data Connectors/CBS/CTM360_CBS_API_functionApp.json

```diff
@@ -0,0 +1,129 @@
+        },
+        {
+            "title": "",
+            "description": " ***important *** Go to ***Azure portal ---> Resource groups ---> [your_resource_group] --->  [appName](type: Storage account) ---> Storage Explorer ---> BLOB CONTAINERS ---> CTM360 ---> Upload*** and create file on your machine named flagCBS.txt and type inside it 'false' and select it for upload (this is done so that to update if the system log the old data to sentinel)\n"
+        },
+        {
+            "title": "",
+            "description": "**Option 2 - Manual Deployment of Azure Functions**\n\n Use the following step-by-step instructions to deploy the 'CyberBlindSpot' connector manually with Azure Functions."
+        },
+        {
+            "title": "1. Create a Function App",
+            "description": "1.  From the Azure Portal, navigate to [Function App](https://portal.azure.com/#blade/HubsExtension/BrowseResource/resourceType/Microsoft.Web%2Fsites/kind/functionapp).\n2. Click **+ Create** at the top.\n3. In the **Basics** tab, ensure Runtime stack is set to **python 3.8**. \n4. In the **Hosting** tab, ensure **Plan type** is set to **'Consumption (Serverless)'**.\n5.select Storage account\n6. 'Add other required configurations'. \n5. 'Make other preferable configuration changes', if needed, then click **Create**."
+        },
+        {
+            "title": "2. Import Function App Code(Zip deployment)",
+            "description": "1. Install Azure CLI\n2. From terminal type **az functionapp deployment source config-zip -g <ResourceGroup> -n <FunctionApp> --src <Zip File>** and hit enter. Set the `ResourceGroup` value to: your resource group name. Set the `FunctionApp` value to: your newly created function app name. Set the `Zip File` value to: `CTM360Connector.zip`(path to your zip file). Note:- Download the zip file from the link - [Function App Code](https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/CTM360/Data%20Connectors/CTM360/CTMConnector.zip)"
```

Comment:
Please correct the instructions, if you want to add cbs function app or ctm360 function app. Also please correct the hyper link which will refer to the raw zip file